applications/ContactStructuralMechanicsApplication/custom_utilities/interface_preprocess.cpp

```diff
@@ -269,7 +269,7 @@ void InterfacePreprocessCondition::CreateNewCondition(
 {
     KRATOS_TRY;
 
-    Condition::Pointer p_cond = Condition::Pointer(rCondition.Create(ConditionId, rGeometry, pThisProperties));
+    Condition::Pointer p_cond = Condition::Pointer(rCondition.Create(ConditionId, rGeometry.Points(), pThisProperties));
```

Comment:
@tteschemacher I would still try to fix this

Reply:
Shall we do this in a new PR?

Reply:
I think the const version of the `pointsArrayType&()` is missing

Reply:
> Shall we do this in a new PR?

yes